docs/content/getting-started/faq.md

```diff
@@ -0,0 +1,114 @@
+# FAQ
+
+## Why is Traefik Answering XXX HTTP Response Status Code ?
+
+Traefik is a dynamic reverse proxy,
+and while the documentation often demonstrates configuration options through file examples,
+the core feature of Traefik is its ability to be provided with configuration dynamically.
+
+The file provider allows providing [dynamic configuration](../configuration-overview/#the-dynamic-configuration) through a file or directory.
+This is different from the [static configuration](../configuration-overview/#the-static-configuration),
+which can also be provided by a file.
+
+For HTTP configurations, the EntryPoint concept matches the Transport Layer (TCP),
+while the Router concept matches the Presentation (TLS) and Application layers (HTTP).
```

Comment:
In addition, the configuration englobes concepts such as the EntryPoint which can be seen as a listener on the Transport Layer (TCP), as apposed to the Router which is more about the the Presentation (TLS) and Application layers (HTTP). And there can be as many routers as one wishes for a given EntryPoint.